### PR TITLE
Make adjustments to the Project Item Issue template

### LINKED ISSUE #253: 🚧Make adjustments to the Project Item Issue template
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_<title_**>' section.

### Description

Make various changes to the **project item issue template.**
The name of the file for the issue template is 👉🏼 **_project-item-issue-template.yml_**

---

### Acceptance Criteria

**This issue is finished when:**
- [x] Remove the _**\<title\>**_ section from the title text box.
image
- [x] Change the text of the checkbox at the top of the template to the value below.
  - I have updated the title and left the 🚧 emoji behind.
image
- [x] Change the smaller gray text below the **Description** section label to the result below
image
- [x] Make changes below to the **Acceptance Criteria** section.
image
- [x] Make the following changes to the **ToDo Items** section.
image
- [x] Fix spelling error in and add text in the **Issue Dependencies** section.
image
- [x] Change the descriptive text in the **Related Work** section to the value below.
image

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [x] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

_No response_

### Related Work

- KinsonDigital/issues-repo#9

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/project-item-issue-template.yml [hunk 1]
```diff
@@ -35,48 +33,48 @@ body:
...
+      No Issue Dependencies Added.
       Example:
         - #10 Create new class
         - #20 Add unit tests to service
         - #30 Cleanup code in UI
     description: |
-      A list of issues that are dependencies of this issue and need to be finishes first
-      Use the '#\<number\>' syntax to link to an issue
+      A list of issues that are dependencies of this issue and need to be finished first.
+      Use the '#\<issue-number\>' syntax to link to an issue.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/project-item-issue-template.yml:54]: Should this say "link to an issue" or link an issue?
```suggestion
      Use the '#\<issue-number\>' syntax to link an issue.
```

File: .github/ISSUE_TEMPLATE/project-item-issue-template.yml [hunk 2]
```diff
@@ -35,48 +33,48 @@ body:
...
+      No Related Work Items Added.
       Example:
         - #10 Create new class
         - #20 Add unit tests to service
         - #30 Cleanup code in UI
     description: |
-      A list of issues that are related to this issue but that is not a dependency
-      Use the '#\<number\>' syntax to link to an issue
+      A list of issues related to this issue but are not a dependency.
+      Use the '#\<issue-number\>' syntax to link to an issue.
```
Comment (kselena) [inline .github/ISSUE_TEMPLATE/project-item-issue-template.yml:69]: Should this say "link to an issue" or link an issue?

```suggestion
      Use the '#\<issue-number\>' syntax to link an issue.
```

**Review by kselena — CHANGES_REQUESTED** [review]

**Review by kselena — APPROVED** [review]
